### PR TITLE
Rubyconf lt

### PR DESCRIPTION
**Pull request by PragTob**
Don't merge this yet (although it's long overdue) - there are a couple of items that need to get in before that:

- [x] new shoes-highlighter release with code highlighting fix shoes/shoes-highlighter#3
- [x] wingtips custom code highlighting needs to land PragTob/wingtips#32
- [x] general settings in wingtips so that they can be used as template settings to get rid of the `tobi_*` templates
- [x] update wingtips version used here
- [x] merge

:)

### CODE REVIEW

File: code/flows.rb [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
   flow do
-    para "One"
-    para "Two"
-    para "Three"
+    banner "One"
+    banner "Two"
+    banner "Three"
+    banner "Four"
+    banner "Five"
+    banner "Six"
```
Comment (PragTob) [inline code/flows.rb]: I didn't really like the "increase para size" hack, I think using something like banner is more sensible - would remove the hack as well if you agree :)

**Comment by PragTob** [comment]
Basically this now waits on PragTob/wingtips#32 and a new wingtips release, otherwise it should be all good if it gets a :+1: from you

doesn't only rely on the code_highliting but also the more generalized options :)

**Comment by PragTob** [comment]
PR has landed, updated wingtips, updated shoes to get the star fill fix and added a note.

Ready to merge from my side, if you think it's good and didn't mess with anything :)

**Comment by jasonrclark** [comment]
Looks great! Might spend my computering time this evening looking at #5 and giving a closer read-through to all your great additions. Thanks for the work on this!

**Pull request merged**